public/index.html

```diff
@@ -5,6 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.53.0/mapbox-gl.css"
```

Comment:
Should we add a comment stating whether this should stay in sync with the version in package.json? With package.json I usually save them as exact versions to make it easier to track what's breaking.

Reply:
I wasn't actually sure why this was here - I think it got added at some other stage so I removed it (vue should add it at build time so it still works).

Reply:
nvm - it's CSS. I moved its import to `main.js` !